Tests/AbstractTestCase.php

```diff
@@ -5,7 +5,10 @@
 use Akeneo\Bundle\BatchBundle\Command\BatchCommand;
 use Akeneo\Test\IntegrationTestsBundle\Configuration\CatalogInterface;
 use Akeneo\Test\IntegrationTestsBundle\Security\SystemUserAuthenticator;
+use Doctrine\ORM\EntityManagerInterface;
```

Comment:
Why do we need new `use` clauses without changes in the class?

Reply:
Because it is used in phpdoc ;)